scripts/create_patches_files.py

```diff
@@ -73,13 +78,27 @@ def create_files(series_name: Dict[str, str], series_url: Dict[str, str], downlo
     with open("./artifact_names.txt", "w") as f:
         f.write(str(artifact_names))
 
+def interesting_patch(patch: Dict[str, Any]):
+    """Grep the patch mbox file for key terms/email addresses."""
+    r = requests.get(patch["mbox"])
+    r.encoding = r.apparent_encoding
```

Comment:
Does this assignment change r.text result? Seems like r.encoding is not used in the function so I assume this is necessary for r.text. Then, I think it's good!

Reply:
It might not be necessary when dealing with the patchworks server - [this answer](https://stackoverflow.com/a/52615216) mentioned it and it appears like r.encoding affects the result of r.text if it's not properly specified.

Reply:
I see. Can we have this link as a comment? It might be helpful for the future readers